transformations/us_state_abbreviation_transformation/README.md

```diff
@@ -0,0 +1,21 @@
+# US State Abbreviation 🦎  + ⌨️ → 🐍
+This perturbation adds US State/Territory/Commonwealth name/abbreviation: Pennsylvania -> PA or PA -> Pennsylvania
+
+Author name: Yiwen Shi
+Author email: yiwen.shi@drexel.edu
+Author Affiliation: Drexel University
+
+Author name: Hualou Liang
+Author email: hualou.liang@drexel.edu
+Author Affiliation: Drexel University
+
+## What type of a transformation is this?
+This transformation acts as a perturbation and makes lexical substitutions. 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document contains US states name or abbreviation as input like text classification, 
```

Comment:
Thanks for the neat PR! However, I worry the scope of the transformation may be a bit constrained. I would suggest consider adding expanders/shorthands for other countries, province, or even other kinds of terms. For example, [this project on scientific term expansion](https://github.com/allenai/scispacy#abbreviationdetector) seems very relevant.

Reply:
Also, have you considered doing the other way around, i.e. abbrev -> full form?